package.json

```diff
@@ -45,6 +45,7 @@
     "deep-diff": "1.0.2",
     "dotenv": "8.2.0",
     "express": "4.17.1",
+    "google-libphonenumber": "^3.2.6",
```

Comment:
We used fix version numbers (in hopefully most our repos) so that they are shared across developer machines and `npm install` doesn't try and bump these

This would be `3.2.6`

Reply:
Ah good catch, thanks.